src/odemis/gui/comp/overlay/world.py

```diff
@@ -390,18 +391,26 @@ def on_roa(self, roa):
         roi (tuple of 4 floats): left, top, right, bottom position relative to the SEM image
 
         """
-        phys_rect = self.convert_roi_ratio_to_phys(roa)
+
+        if self._scanner:
+            phys_rect = self.convert_roi_phys_to_ratio(roa)
+        else:
+            phys_rect = roa
+
```

Comment:
Please don't add double empty lines. Just one line is enough. Also, avoid starting the function with an empty line.